Tools/PowershellModule/src/Library/New-ARMObjects.ps1

```diff
@@ -4,29 +4,29 @@ Function New-ARMObjects
 {
     <#
     .SYNOPSIS
-    Creates the Azure Resources to stand-up a Windows Package Manager Rest Source.
+    Creates the Azure Resources to stand-up a Windows Package Manager REST Source.
 
     .DESCRIPTION
     Uses the custom PowerShell object provided by the "New-ARMParameterObject" cmdlet to create Azure resources, and will 
-    create the the Key Vault secrets and publish the Windows Package Manager rest source rest apis to the Azure Function.
+    create the the Key Vault secrets and publish the Windows Package Manager REST source REST apis to the Azure Function.
         
     The following Azure Modules are used by this script:
-        Az.Resources --> Invoke-AzResourceAction
-        Az.Accounts  --> Connect-AzAccount, Get-AzContext
-        Az.Websites  --> Get-AzWebapp
-        Az.Functions --> Get-AzFunctionApp
+        Az.Resources
+        Az.Accounts
+        Az.Websites
+        Az.Functions
 
     .PARAMETER ARMObjects
     Object returned from the "New-ARMParameterObject" providing the paths to the ARM Parameters and Template files.
 
     .PARAMETER RestSourcePath
-    Path to the compiled Function ZIP containing the Rest APIs
+    Path to the compiled Function ZIP containing the REST APIs
 
     .PARAMETER AzResourceGroup
     Resource Group that will be used to create the ARM Objects in.
 
     .EXAMPLE
-    New-ARMObjects -ARMObjects $ARMObjects -RestSourcePath "C:\WinGet-CLI-RestSource\CompiledFunction.zip" -AzResourceGroup "WinGetResourceGroup"
+    New-ARMObjects -ARMObjects $ARMObjects -RestSourcePath "C:\WinGet-CLI-RestSource\WinGet-CLI-RestSource.zip" -AzResourceGroup "WinGet"
```

Comment:
What is this path and function name. it doesnt align with the zip file we publish. What are you trying to point to here.

Reply:
Release artifact (downloaded & Unziped) on the computer.